astroquery/noirlab/core.py

```diff
@@ -90,12 +102,138 @@ def query_region(self, coordinate, radius=0.1, cache=True):
         """
         self._validate_version()
         ra, dec = coordinate.to_string('decimal').split()
-        url = f'{self.url}?POS={ra},{dec}&SIZE={radius}&format=json'
+        url = f'{self.siaurl}?POS={ra},{dec}&SIZE={radius}&format=json'
         response = self._request('GET', url,
                                  timeout=self.TIMEOUT,
                                  cache=cache)
         response.raise_for_status()
         return astropy.table.Table(data=response.json())
 
+    # #!def query_region(self, coordinate, radius=0.1, cache=True):
+    # #!    """Query for NOIRLab observations by region of the sky.
+    # #!
+    # #!    Given a sky coordinate and radius, returns a `~astropy.table.Table`
+    # #!    of NOIRLab observations.
+    # #!
+    # #!    Parameters
+    # #!    ----------
+    # #!    coordinates : str or `~astropy.coordinates` object
+    # #!        The target region which to search. It may be specified as a
+    # #!        string or as the appropriate `~astropy.coordinates` object.
+    # #!    radius : str or `~astropy.units.Quantity` object, optional
+    # #!        Default 0.1 degrees.
+    # #!        The string must be parsable by `~astropy.coordinates.Angle`.The
+    # #!        appropriate `~astropy.units.Quantity` object from
+    # #!        `~astropy.units` may also be used.
+    # #!
+    # #!    Returns
+    # #!    -------
+    # #!    response : `~astropy.table.Table`
+    # #!    """
+    # #!    self._validate_version()
+    # #!    ra, dec = coordinate.to_string('decimal').split()
+    # #!    url = f'{self.siaurl}?POS={ra},{dec}&SIZE={radius}&format=json'
+    # #!    response = self._request('GET', url,
+    # #!                             timeout=self.TIMEOUT,
+    # #!                             cache=cache)
+    # #!    response.raise_for_status()
+    # #!    return astropy.table.Table(data=response.json())
```

Comment:
your `_parse_result` function can pretty much take this `response` and do this one line - i.e., make a table from the json.

The idea of the `query_<blah>_async` <-> `query_<blah>` is to have the remote query component in one function and the parse-the-response-into-a-user-readable-table part in a separate function

Reply:
That was commented out code because I couldn't be sure of how the async API is supposed to work (really could use more documentation over for how the "async_to_sync" decorator is supposed to be used. Ultimately, James Dempsey gave me more useful help by asking my specific questions.  Since i **_think_** I'm using async ok, I've removed the commented out code.

Reply:
OK, you can just delete this commented out code, then.